Automattic/wp-calypso
# Jetpack Backup, Jetpack Scan: Standardize upsell look/feel for incompatible sites

#### Changes proposed in this Pull Request

* Add new VaultPress logo with less padding to shared `assets/images/jetpack` directory
* In `HasVaultPressSwitch`, check Scan state as well as Rewind state. (+ unit tests)
* Attempt to standardize icon sizes and page layouts between Backup and Scan upsells.
* Attempt to standardize copy between Backup and Scan upsells for VaultPress.
* Move `ScanUpsellPage` into its own directory, so as to match `BackupUpsellPage`.

Fixes `p6TEKc-45c-p2`.

#### Implementation notes

* In the future, we may want to come back and see if our upsells can be DRYed up a bit. I'm noticing a lot of duplication, and possibly some stale CSS rules we can get rid of.

#### Testing instructions

**Prerequisites:** You'll need access to a VaultPress site and the primary site of a multi-site network.

* Load this pull request in your test environment, in both Calypso and Jetpack Cloud. Most of the changes here are style-related, so make sure you test everything in both mobile and desktop layouts.
* Load the Backup and the Scan page, then verify for the following scenarios:
  * Regular site, but no Backup/no Scan: upsell is visible, upsell matches production
  * Backup-only: upsell is visible for Scan but not Backup, both pages' behavior match production
  * Scan-only: upsell is visible for Backup but not Scan, both pages' behavior match production
  * Multi-site: upsell is visible, matches the screenshots below
  * VaultPress: upsell is visible, matches the screenshots below


#### Before-and-after screenshots

##### Calypso mobile: Before above, After below

![wpcom_mobile](https://user-images.githubusercontent.com/670067/96775550-4bfbdf80-13ad-11eb-9729-3e036c11e83b.png)

##### Jetpack Cloud mobile: Before above, After below

![cloud_mobile](https://user-images.githubusercontent.com/670067/96775572-54ecb100-13ad-11eb-948c-65a40da647ed.png)

##### Calypso desktop: Before left, After right

![wpcom_desktop](https://user-images.githubusercontent.com/670067/96776025-0a1f6900-13ae-11eb-87e8-a66bcb2a29ed.png)

##### Jetpack Cloud desktop: Before left, After right

![cloud_desktop](https://user-images.githubusercontent.com/670067/96776661-f1638300-13ae-11eb-9129-4b060b9b94f7.png)
